Celeste.Mod.mm/Mod/UI/OuiModOptions.cs

```diff
@@ -41,9 +41,66 @@ public interface ISubmenu { }
+                                List<EverestModuleMetadata> missingDependenciesForMod = mod.Item1.Dependencies
+                                    .FindAll(dep => !Everest.Loader.DependencyLoaded(dep));
+                                missingDependencies.AddRange(missingDependenciesForMod);
+
+                                if (missingDependenciesForMod.Count != 0) {
+                                    // format their names and versions, and join all of them in a single string
+                                    missingDepsString = string.Join(", ", missingDependenciesForMod.Select(dependency => dependency.Name + " | v." + dependency.VersionString));
+
+                                    // ensure that string is not too long, or else it would break the display
+                                    if (missingDepsString.Length > 40) {
+                                        missingDepsString = missingDepsString.Substring(0, 40) + "...";
+                                    }
+
+                                    // wrap that in a " ({list} not found)" message
+                                    missingDepsString = $" ({missingDepsString} {Dialog.Clean("modoptions_coremodule_notloaded_notfound")})";
+                                }
+                            }
+
+                            menu.Add(new TextMenuExt.SubHeaderExt(mod.Item1.Name + " | v." + mod.Item1.VersionString + missingDepsString) {
+                                HeightExtra = 0f,
+                                TextColor = Color.PaleVioletRed
+                            });
+                        }
+                    }
+                }
+
+                if (Everest.Updater.HasUpdate) {
+                    menu.Add(new TextMenu.Button(Dialog.Clean("modoptions_coremodule_update").Replace("((version))", Everest.Updater.Newest.Build.ToString())).Pressed(() => {
+                        Everest.Updater.Update(OuiModOptions.Instance.Overworld.Goto<OuiLoggedProgress>());
+                    }));
+                }
+
+                if (missingDependencies.Count != 0) {
+                    menu.Add(new TextMenu.Button(Dialog.Clean("modoptions_coremodule_downloaddeps")).Pressed(() => {
+                        OuiDependencyDownloader.MissingDependencies = missingDependencies;
+                        OuiModOptions.Instance.Overworld.Goto<OuiDependencyDownloader>();
+                    }));
+                }
+            }
+
             List<EverestModule> modules = new List<EverestModule>(Everest._Modules);
             if (Everest.Loader._ModOptionsOrder != null && Everest.Loader._ModOptionsOrder.Count > 0) {
                 foreach (string modName in Everest.Loader._ModOptionsOrder) {
+                    //Hack to allow user to position Everest Core in ModSettings
+                    if (modName.ToLower().Equals("everest")) {
+                        modules.Find(mod => mod.Metadata.Name == "Everest").CreateModMenuSection(menu, inGame, snapshot);
```

Comment:
Everest doesn't get removed from the `modules` list, so the Everest Core options will get rendered twice, won't they?